velox/docs/configs.rst

```diff
@@ -414,6 +414,16 @@ Each query can override the config by setting corresponding query session proper
      - bool
      - true
      - If true, the partition directory will be converted to lowercase when executing a table write operation.
+   * - allow-null-partition-keys
+     - allow_null_partition_keys
+     - bool
+     - true
+     - Determines whether null values for partition keys are allowed or not. if false, it will threw error message
+       like "Partition key must not be null" when writing data with null partition key.
+       **Important Note:**
```

Comment:
This information is important, but not more important as other information in this doc. Let's remove this header. It is too loud.

Reply:
good advice, fixed.